src/parser/tokens.rs

```diff
@@ -156,6 +156,10 @@ impl Token {
         Token::specific_token(input, "else")
     }
 
+    pub fn jk_return(input: &str) -> ParseResult<&str, &str> {
```

Comment:
```suggestion
    pub fn return_tok(input: &str) -> ParseResult<&str, &str> {
```
Nit but just to stay in line with the other token functions